packages/react-dom/unstable_testing.js

```diff
@@ -7,7 +7,30 @@
  * @flow
  */
 
-export * from './index.js';
+export {
+  createPortal,
+  findDOMNode,
+  flushSync,
+  hydrate,
+  render,
+  unmountComponentAtNode,
+  unstable_batchedUpdates,
+  unstable_createEventHandle,
+  unstable_renderSubtreeIntoContainer,
+  unstable_runWithPriority, // DO NOT USE: Temporarily exposed to migrate off of Scheduler.runWithPriority.
+  useFormStatus,
+  useFormState,
+  prefetchDNS,
+  preconnect,
+  preload,
+  preloadModule,
+  preinit,
+  preinitModule,
+  version,
+} from './index.js';
+
+export {createRoot, hydrateRoot} from './client.js';
```

Comment:
Heads up @gnoff, I needed to update the imports in the unstable_testing entrypoints so using create/hydrateRoot doesn't error. Let me know if there are any concerns, but I think this is what we want to do, at least while this is unstable.